targets/particle/Makefile.particle

```diff
@@ -0,0 +1,78 @@
+# Copyright 2016 Samsung Electronics Co., Ltd.
+# Copyright 2016 University of Szeged
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PARTICLE_FIRMWARE := $(CURDIR)/../particle/firmware
+
+JERRYDIR ?= $(CURDIR)
+JERRYHEAP  ?= 16
+
+BUILD_DIR  ?= $(JERRYDIR)/build/particle
+
+EXT_CFLAGS += -mlittle-endian -mthumb -mcpu=cortex-m4
+EXT_CFLAGS += -Wno-error=format=
+
+.PHONY: jerrycore jerry-main flash clean
+
+PARTICLE_BUILD_CONFIG = \
+	 INCLUDE_DIRS=$(JERRYDIR)/jerry-core \
+	 LIBS=jerry-core \
+	 PLATFORM=photon \
+	 LIB_DIRS=$(BUILD_DIR)/lib \
+	 APPDIR=$(JERRYDIR)/targets/particle/source \
+	 TARGET_FILE=jerry_main \
+	 TARGET_DIR=$(BUILD_DIR) \
+	 LDFLAGS=--specs=nano.specs
+
+all: jerrycore jerry-main
+
+jerrycore:
+	mkdir -p $(BUILD_DIR)
+	cmake -B$(BUILD_DIR) -H./ \
+	 -DJERRY_LIBC=OFF \
+	 -DJERRY_LIBM=OFF \
```

Comment:
As far as I can remember, some (one or two) math routines are used even in case of the minimal profile. However, I cannot find any `-lm` switches in the build. What do I miss?